src/utp_stream.cpp

```diff
@@ -799,6 +802,23 @@ void utp_socket_drained(utp_socket_impl* s)
 	s->maybe_trigger_send_callback();
 }
 
+boost::int32_t adjust_receive_buf_size(boost::uint32_t const& rtt)
```

Comment:
the ``rtt`` should not be taken by ``const&`` here, just by value (but it still makes sense to declare it ``const``)